client/landing/stepper/declarative-flow/internals/steps-repository/site-creation-step/index.tsx

```diff
@@ -56,7 +56,12 @@ const SiteCreationStep: Step = function SiteCreationStep( { navigation, flow } )
 	let siteVisibility = Site.Visibility.PublicIndexed;
 
 	// Link-in-bio flow defaults to "Coming Soon"
-	if ( isLinkInBioFlow( flow ) || isFreeFlow( flow ) || isMigrationFlow( flow ) ) {
+	if (
+		isLinkInBioFlow( flow ) ||
+		isFreeFlow( flow ) ||
+		isMigrationFlow( flow ) ||
+		isCopySiteFlow( flow )
+	) {
```

Comment:
Interestingly, I don't think this is quite enough...

My site copy did end up with the Coming Soon banner:

<img width="1892" alt="image" src="https://user-images.githubusercontent.com/36432/212895302-96bf0192-42c5-4845-ba40-931a6c994599.png">

However, 'Privacy' seems to be be set as public:

<img width="1892" alt="image" src="https://user-images.githubusercontent.com/36432/212895417-10c9145e-e778-449b-a4b1-5f7cce025401.png">

Maybe there's some other bit of data that needs to be changed?

Reply:
In my case, it worked similarly. I did the following:
* I clicked "Log in" on the Coming Soon frontend to get automatically logged in
* I refreshed the General Settings page, and it displayed the Coming Soon status correctly.

Maybe we have two other issues here - auto login is not working automatically on cloned sites, and the redux store is not refreshed correctly without explicit page refresh?

Reply:
@wojtekn Was your original site set to 'Coming Soon' or 'Public'? Mine was 'Public'

Reply:
In the initial flow, it was "Coming Soon". I've just checked it for public source site and it works in the same way in both cases.

Reply:
We figured it out:

> While the Simple shadow site is set to 'Coming Soon', the clone of the Atomic site can be 'Public' for a brief period until the post-transfer job is run. This can cause an unexpected user experience in Calypso where the user sees their site as 'Public' in the backend but it's Coming Soon' in the frontend.

Created https://github.com/Automattic/dotcom-forge/issues/1510 for following up on that.